fighthealthinsurance/templates/categorize.html

```diff
@@ -1,21 +1,32 @@
 {% extends 'base.html' %}
 
 {% block title %}
-Categorize your denial (so we can generate the right kind of appeal)
+Categorize Your Denial
```

Comment:
You'll probably need to update the selenium test because we assert titles as we go.